### PR TITLE
added *args to super.__init__

### PR DESCRIPTION
**Pull request by maartenschermer**
Passing *args to the PurePosixPath.

### CODE REVIEW

File: ibridges/utils/path.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class IrodsPath(pathlib.PurePosixPath):
 
     def __init__(self, session, *args, **kwargs):
         self.session = session
-        super().__init__()
+        super().__init__(*args)
```
Comment (chStaiger) [inline ibridges/utils/path.py:26]: Is the **kwargs not needed here?

**Review by chStaiger — APPROVED** [review]
Just one question to answer ;)
Thanks for finding the bug.

**Comment by qubixes** [comment]
This doesn't work for me, I get 
`TypeError: object.__init__() takes exactly one argument (the instance to initialize)`.

**Comment by qubixes** [comment]
@maartenschermer Which version of python are you using? I think we should just drop the inheritance..

**Comment by maartenschermer** [comment]
@qubixes I'm running Python 3.12. Maybe I should downgrade.
@chStaiger the __init__ of the paren only accepts *args

**Comment by qubixes** [comment]
@maartenschermer I'm running 3.11 and it doesn't work for me. I saw that they have made a lot of changes from 3.11 to 3.12, so I'm afraid we'll have to write our own implementation..

**Comment by qubixes** [comment]
@maartenschermer I'll start working on a new implementation of pathlib for the IrodsPath.

**Comment by chStaiger** [comment]
Then I will reject the this PR and w e will wait for a new better version of IrodsPath.

**Pull request closed without merging**